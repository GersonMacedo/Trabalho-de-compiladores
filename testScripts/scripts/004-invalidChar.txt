! -s LEXICAL
! Should stop at the first invalid char outside the comments
!'"@#$%&{}[]{}|\?
'"@#$%&{}[]{}|\?