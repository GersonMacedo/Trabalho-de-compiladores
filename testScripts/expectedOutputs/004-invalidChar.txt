
Performing only lexical analysis at testScripts/scripts/004-invalidChar.txt
Scanner::Invalid char ''' at line 4 column 1
